templates/dart/lib/id.dart.twig

```diff
@@ -4,13 +4,34 @@ part of {{ language.params.packageName }};
 class ID {
   ID._();
 
-  /// Have Appwrite generate a unique ID for you.
-  static String unique() {
-    return 'unique()';
+  // Generate an hex ID based on timestamp
+  // Recreated from https://www.php.net/manual/en/function.uniqid.php
+  static String _hexTimestamp() {
```

Comment:
Seems to work, I think the timestamp behavior here is a little different from Kotlin.